src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -113,18 +115,18 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
                         bigpay_token: {
                             token: paymentData.instrumentId,
                         },
-                        verification_value: '000',
                         credit_card_token: {
                             token: JSON.stringify({
                                 checkoutId: this._digitalRiverCheckoutData.checkoutId,
                             }),
                         },
+                        set_as_default_stored_instrument: shouldSetAsDefaultInstrument || null,
                     },
-                    set_as_default_stored_instrument: shouldSetAsDefaultInstrument,
                 },
             };
 
-            return await this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload));
+            return this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload))
+                .catch(error => this._processAdditionalAction(error, shouldSetAsDefaultInstrument, paymentData.instrumentId, methodId));
```

Comment:
This seems like an exact copy of the new submit, what do you think about abstracting it as a `submitVaultedInstrument()`?

```tsx
if (isVaultedInstrument()) {
  try {
    return await this._submitVaultedInstrument(instrumentId);
  } catch (error) {
    if (!isAuthenticateSourceAction(error)) {
      throw error;
    }

    await this._authenticateSource();

    return await this._submitVaultedInstrument(instrumentId);
}
```